shared/reducers/chat2.tsx

```diff
@@ -846,10 +846,11 @@ const reducer = Container.makeReducer<Actions, Types.State>(initialState, {
         }
         // We might have a placeholder for this message in there with ordinal of its own ID, let's
         // get rid of it if that is the case
-        if (message.id) {
```

Comment:
there are flows where message.id is zero but existing does have the id

Reply:
I don't see how this can be right, it seems like this has been changed to clear this placeholder on any kind of message that comes up here at all.

Reply:
Also the behavior I see if you tap into a thread and placeholders are stuck there, and the real messages are not there. This fix seems like it is trying to fix a problem where both the real message and placeholder are there .